collection/pass-manager/steal-keepass-passwords-using-keefarce.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: steal KeePass passwords using KeeFarce
+    namespace: collection/pass-manager
+    author: "@Ana06"
+    scope: file
+    att&ck:
+      - Credential Access::Credentials from Password Stores::Password Managers [T1555.005]
+    references:
+      - https://github.com/denandz/KeeFarce
+      - https://keepass.info/help/kb/sec_issues.html
+    examples:
+      - 1e609dffd12a59ea5d5c9b3055939b1f
```

Comment:
This file seems to be the code in https://github.com/denandz/KeeFarce/tree/5276c9253f5b79d83fded81782a1ae11dc21bb59/src/KeeFarce (or a version of it)

Reply:
please add the file to capa-testfiles

Reply:
> Do we need to detect https://github.com/denandz/KeeFarce/tree/master/src/BootstrapDLL64 (injected DLL) in this rule as well? For example with 225ba611b727ac7e25b53615b969050a

I was waiting for an answer to this before uploading the sample/s (to know if I should upload one or two).

Reply:
whatever you think makes more sense, I'm fine capturing the injecting binary only for now

Reply:
Merging this only detecting the injecting binary. If someone else has a different opinion, please let me know and I can extend the rule 😉